packages/react-compose-next/README.md

```diff
@@ -0,0 +1,3 @@
+# @fluentui/react-compose-next
+
+**TBD**
```

Comment:
I will add docs in a follow up PR

Reply:
I think we should fill this out with what we have working.  Let's keep merges as user-ready as possible.

Reply:
I added initial docs for `compose()` function. Feel free to update it if I missed something